### DISCUSSION TITLE
v0.5.4

### DISCUSSION THREAD

**Discussion by dail8859** [dail8859/NotepadNext#187] [New Releases]
## What's Changed
* Add Debug Log dock for displaying debug output in the application
* Add command line argument `--reset-settings` to clear all application settings
* Add basic context menu to Search Results
* Add larger icon for Start Menu on Windows
* Add automatic Winget releases
* Select matching text in editor when navigating from search results
* Better codec detection and conversion
* Enable some NSIS Components by default
* Fix memory leak
* Fix line numbers displayed in search results
* Fix "Open With" command on MacOS
* Prevent crash when attempting to show autocomplete off-screen
* Fix missing translation when running from the right-click context menu on Windows

**Full Changelog**: https://github.com/dail8859/NotepadNext/compare/v0.5.3...v0.5.4

<hr /><em>This discussion was created from the release <a href='https://github.com/dail8859/NotepadNext/releases/tag/v0.5.4'>v0.5.4</a>.</em>

**Comment by pp55000** [comment]
Looking forward to updating as soon as possible!!!

**Comment by pp55000** [comment]
When to update the new version？senior

**Comment by dail8859** [comment]
v0.5.4 is already available. I haven't had time to create another release yet.

**Comment by phildeng** [comment]
什么时候支持Mac m芯片

**Reply by dail8859** [reply]
See #199
